src/edge/ssr/crelox.py

```diff
@@ -0,0 +1,41 @@
+from edge.ssr import rc, _c, Reaction, Integration, Excision, Inversion, RMCE
+
+
+class Sites(object):
+    loxP     = _c("ATAACTTCGTATA GCATACAT TATACGAAGTTAT")
+    lox66    = _c("ATAACTTCGTATA GCATACAT TATACGAACGGTA")
+    lox71    = _c("TACCGTTCGTATA GCATACAT TATACGAAGTTAT")
+    lox72    = _c("TACCGTTCGTATA GCATACAT TATACGAACGGTA")
+
+    lox5171  = _c("ATAACTTCGTATA GtAcACAT TATACGAAGTTAT")
+    lox2272  = _c("ATAACTTCGTATA GgATACtT TATACGAAGTTAT")
```

Comment:
Why are some of these bases lowercase?

Reply:
those are the mutated sequences from the WT, i am trying to highlight the differences